### DISCUSSION TITLE
A newcomer who's a little lost

### DISCUSSION THREAD

**Discussion by t-rosa** [chakra-ui/zag#1547] [Q&A] [answered]
Hello, I'm a bit lost!

For context: I've just migrated my React app with Tailwind and Radix/Shadcnui to Tailwind with Ark/ParkUI and I'm worried about several things.

Will the library and ecosystem be well maintained over time?

Will the library take advantage of the future React compiler?

What's the difference between Zag, Ark and Park?

At the moment I'm using Park but I'm wondering if I shouldn't just use Zag with my own styles, also Zag documentation seems more advanced than the other two.

So, finally, I'm looking to see if I made the right bet by migrating to this technology.

Thank you in advance for your answers.

**Comment by segunadebayo** [comment, marked as answer]
Ultimately, the choice is yours to pick what works for you.

> Will the library and ecosystem be well maintained over time?
- Zag.js will remain maintained over time.

> Will the library take advantage of the future React compiler?
- My intuition is that the React compiler is for optimizing application-level code, not library-level code. I could be wrong, though. For the most part, Zag.js doesn't depend too much on React primitives to work, which is the goal of the project.

> What's the difference between Zag, Ark, and Park?
- Zag is a framework agnostic base that models common UI patterns as a finite state machine. Ark converts Zag to a "Radix-like" experience so it's more approachable. Park is a styled version of Ark.

Using Ark is just fine for your use case.

**Reply by t-rosa** [reply]
Thank you for your reply.

I took the time to read the docs in more detail and the role of each library is much clearer (I should have done this directly).

After a few weeks of use, I find Ark more solid and viable than the alternatives.

The fact that Zag.js will be maintained over time confirms my choice.
